### PR TITLE
chore: add prettier-eslint formatter.

### PR DESCRIPTION
**Pull request by anthogez**
- [x] Tests written and linted
- [x] Documentation written / README.md updated [https://snyk.io/docs/snyk-for-node/](i)
- [x] Follows [CONTRIBUTING agreement](CONTRIBUTING.md)
- [x] Commit history is tidy [https://git-scm.com/book/en/v2/Git-Branching-Rebasing](i)
- [x] Reviewed by Snyk team

### What this does
add prettier-eslint formatter.

### CODE REVIEW

**Comment by dkontorovskyy** [comment]
Can you separate into 2 commits adding prettier and prettier changes, please?

File: package.json [hunk 1]
```diff
@@ -42,8 +43,13 @@
     "@types/lodash": "^4.14.136",
     "@types/node": "^6.14.7",
     "@types/uuid": "^3.4.4",
+    "@typescript-eslint/eslint-plugin": "^2.29.0",
+    "@typescript-eslint/parser": "^2.29.0",
+    "eslint": "6.8.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "prettier": "^1.19.1",
```
Comment (dkontorovskyy) [inline package.json:51]: prettier is now on version `2`. Why are we in favor of using version `1`?

Comment (anthogez) [reply package.json:51]: Prettier 2 works only with >= 10

**Comment by dkontorovskyy** [comment]
Can you change this commit message to be `chore` `style: files formatted.` as we don't use style commit prefix. It's probably also `chore`. 🙏 
Other than that, great job 💪

**Comment by anthogez** [comment]
> Can you change this commit message to be `chore` `style: files formatted.` as we don't use style commit prefix. It's probably also `chore`. 🙏
> Other than that, great job 💪

Thanks, I used `style` because I saw it available in our documentation

**Pull request merged**